packages/docs/docs/learn/06-recipes/02-variants.mdx

```diff
@@ -153,8 +153,8 @@ function Button({
 }
 
 // Usage
-<Button color="primary" size="medium">Primary</Button>
-<Button color="secondary">Secondary</Button>
+<Button color="primary" size="medium" disabled>Primary</Button>
```

Comment:
There is no need to mark these buttons as disabled. The `disabled` prop is explicitly shown to be optional.